build-system/tasks/extension-helpers.js

```diff
@@ -356,22 +356,28 @@ function buildExtension(
   if (options.compileOnlyCss && !hasCss) {
     return Promise.resolve();
   }
+  const path = 'extensions/' + name + '/' + version;
+
   // Use a separate watcher for extensions to copy / inline CSS and compile JS
   // instead of relying on the watcher used by compileUnminifiedJs, which only
   // recompiles JS.
-  const path = 'extensions/' + name + '/' + version;
-  const optionsCopy = Object.create(options);
```

Comment:
any reason why we removed the copy?

Reply:
The copy was being used by `buildExtension` to implicitly tell the `compileUnminifiedJs` not to use its own watcher during the first build (by setting `optionsCopy.watch` to false).

This is no longer necessary because we're now explicitly passing in this info by setting the `options.continueOnError` field, which has the desired effect during the first build and during re-builds.